senza/cli.py

```diff
@@ -182,13 +197,7 @@ def watching(w: bool, watch: int):
 # and start with an alpha character. Maximum length of the name is 255 characters.
 STACK_NAME_PATTERN = re.compile(r'^[a-zA-Z][a-zA-Z0-9-]*$')
 VERSION_PATTERN = re.compile(r'^[a-zA-Z0-9]+$')
-
-
-def validate_version(ctx, param, value):
-    if not VERSION_PATTERN.match(value):
-        raise click.BadParameter('Version must satisfy regular expression pattern "[a-zA-Z0-9]+"')
-    return value
-
+REGION_PATTERN = re.compile(r'^(eu|ap|us|sa)-(west|east|central|southeast|northeast)-(1|2)$')
```

Comment:
The problem with this approach is that it will fail if Amazon adds an extra region to a place that already as 2 or some region in Africa for example.